tests/unit/wingsail/common/test_lut.py

```diff
@@ -0,0 +1,53 @@
+# import pytest
+
+import numpy as np
+from scipy import interpolate
+
+from controller.common.lut import LUT
+
+
+class TestLUT:
+    # Intialize lookup table
+    look_up_table = [[50000, 5.75], [100000, 6.75], [200000, 7], [500000, 9.75], [1000000, 10]]
+
+    def test_LUT_constructor(self):
+        # set up
+        testLUT = LUT(self.look_up_table)
+
+        # test that LUT return a known value
+        assert testLUT(50000) == 5.75
+
+    def test_unknown_interpolation_exception(self):
+        try:
+            testLUT = LUT(self.look_up_table, "gabagool")
+            assert False  # failure: constructor accepted deli meat interpolation method
+
+        except ValueError:
+            assert True
+
+        except:
+            assert False  # failure: constructor threw wrong exception
```

Comment:
When testing for exceptions, you want to use a context block with the [`pytest.raises`](https://docs.pytest.org/en/stable/reference/reference.html#pytest.raises) function to ensure a code block throws an exception.

Reply:
done. 
```
    def test_unknown_interpolation_exception(self):
        with pytest.raises(ValueError):
            testLUT = LUT(self.look_up_table, "gabagool")

    def test_invalid_table_exception(self):
        invalid_table1 = [[10000, 10000, 10000], [1, 1, 1]]
        invalid_table2 = [10000, 10000, 10000]
        invalid_table3 = [[0, 1], 10000, 10000]

        with pytest.raises(ValueError):
            testLUT = LUT(invalid_table1)

        with pytest.raises(ValueError):
            testLUT = LUT(invalid_table2)

        with pytest.raises(ValueError):
            testLUT = LUT(invalid_table3)
```